nautobot/dcim/factory.py

```diff
@@ -61,18 +61,33 @@
     "F5",
     "Force10",
     "Fortinet",
+    "HP",
     "Huawei",
     "Juniper",
-    "HP",
     "Palo Alto",
 )
 
-# For `Platform.napalm_driver`, either randomly choose based on Manufacturer slugified name.
+# For `Platform.napalm_driver` and `Platform.network_driver`, either randomly choose based on Manufacturer name.
 NAPALM_DRIVERS = {
-    "arista": ["eos"],
-    "cisco": ["ios", "iosxr", "iosxr_netconf", "nxos", "nxos_ssh"],
-    "juniper": ["junos"],
-    "palo-alto": ["panos"],
+    "Arista": ["eos"],
+    "Cisco": ["ios", "iosxr", "iosxr_netconf", "nxos", "nxos_ssh"],
+    "Juniper": ["junos"],
+    "Palo Alto": ["panos"],
```

Comment:
Looks like `napalm_driver` logic (line 172/187) needs to remove call to `slugify()` to match this change.